client/src/nl/sensorlab/videowall/animation/baseanimations/bouncypixels/BouncyPixel.java

```diff
@@ -0,0 +1,92 @@
+package nl.sensorlab.videowall.animation.baseanimations.bouncypixels;
+
+import java.util.ArrayList;
+
+import nl.sensorlab.videowall.animation.BaseAnimation;
+import processing.core.PGraphics;
+import processing.core.PVector;
+
+public class BouncyPixel {
+
+	BouncyPixelSystem parent;
```

Comment:
Again: member visibility: parent has no visibility declared, do all other members need to be public?